rsmtool/test_utils.py

```diff
@@ -138,7 +139,7 @@ def do_run_summary(source, config_file):
     run_summary(config_file, experiment_dir)
 
 
-def check_csv_output(csv1, csv2):
+def check_csv_output(csv1, csv2, file_format='csv'):
```

Comment:
Should this really be called `check_csv_output` if `csv` is only one possible extension?

Reply:
Yeah, I was debating this. :)

Changing them is okay with me. Suggestions on naming convention? Should we just replace "csv" with "file" or is there something that makes more sense?

Reply:
`check_generated_output`?